src/app/config.py

```diff
@@ -22,4 +22,9 @@
 JWT_ENCODING_ALGORITHM = "HS256"
 
 SUPERUSER_LOGIN: str = os.getenv("SUPERUSER_LOGIN")
+if SUPERUSER_LOGIN is None:
+    raise ValueError("SUPERUSER_LOGIN is missing from environment variables")
+
 SUPERUSER_PWD: str = os.getenv("SUPERUSER_PWD")
+if SUPERUSER_PWD is None:
+    raise ValueError("SUPERUSER_PWD is missing from environment variables")
```

Comment:
Since both of them are to be specified, perhaps we could use the minimal
`if SUPERUSER_LOGIN is None or SUPERUSER_PWD is None:`

And about the error, something more suggestive of how to solve this perhaps? Like "Please set 'SUPERUSER_LOGIN' and 'SUPERUSER_PWD' in your environment variables.

Reply:
Yes I was hesitating with the or statement. 

Okay !